signature-specification.md

```diff
@@ -222,10 +222,10 @@ The `keyUsage` extension MUST be present and MUST be marked critical. Bit positi
 
 #### Other requirements
 
+1. Valid certificate chain MUST contain a root certificate. If the certificate chain contains only a root certificate then the root certificate MUST meet [Leaf Certificates](#leaf-certificates) requirements and ignore [Root and Intermediate CA Certificates](#root-and-intermediate-ca-certificates) requirements.
```

Comment:
To be clear, this means self signed cert cannot have the `cA` field as`true`, correct?

Reply:
That's correct.